### DISCUSSION TITLE
Read log after fail

### DISCUSSION THREAD

**Discussion by V3sp** [deployphp/deployer#2900] [Help needed]
Hey there,
I'm looking for the help with getting the log (or verbose) when the deploy will fail.
No-w i use teams integration and i'm trying to make something like:
on fail -> read log file -> set message for teams -> teams:notify:failure 

But i stop on reading the log file becasue log file not exists. Its create afrer deployer end ( i use flag --log)

May be there is a better way to do something like this ?

**Comment by antonmedv** [comment]
Use a wrapper script to do something like this:
```
dep deploy --log
if failed {
  dep notify_failure
}
```
